options_test.go

```diff
@@ -590,3 +615,47 @@ func TestGetBlockDevices(t *testing.T) {
 		})
 	}
 }
+
+func TestGetSocketPath(t *testing.T) {
+	cases := []struct {
+		name        string
+		setup       func()
```

Comment:
Can you modify this to be `setup func() func()` so that the setup function can return a function that is used to clean up?